src/components/client/index/sections/ActiveCampaignsSection/ActiveCampaignCard/ActiveCampaignCard.tsx

```diff
@@ -63,19 +47,35 @@ export default function ActiveCampaignCard({ campaign, index }: Props) {
             : targetAmount.split('.')[0]}
         </Sum>
       </SumWrapper>
+      <Grid
+        sx={{
+          background: `url(${campaignImagesUrl})`,
+          height: '100%',
+          backgroundSize: 'cover',
```

Comment:
Won't this approach cause some images to be either over-stretched or cut from the frame? I think a  better approach would be to set the width to `100%` and then use the [aspectRatio CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio), to adjust the image's height.